Security_Analytics/auditd_analysis/README.md

```diff
@@ -0,0 +1,174 @@
+# Alerting on Auditd CEF Data
+
+This **Getting Started with Elastic Stack** example provides sample files to ingest, analyze and alert on **Auditd Logs in the CEF Format** using the Elastic Stack. 
+
+Included are example Watches for proactively monitoring this data for possible security incidents.  These examples support the Security Analytics blog post series, specifically:
+ 
+[Integrating Elasticsearch with ArcSight SIEM - Part 5](https://elastic.co/blog/integrating-elasticsearch-with-arcsight-siem-part-5).
+
+This example complements the above blog post, providing a means to detect unusual processes starting on servers in the last N minutes. Unusual is defined as "A new process signature that has not occurred on the server historically".
+
+This example relies on the user creating a `process_signature` which acts as a unique identifier for a process. This is an imprecise process, relying on the user selecting fields to concatenate which best represent a unique identifier across server instances when combined.  Too coarse identifier, which fails to encode sufficient properties of the processes characteristics, will result in the alert failing to detect new process effectively. Too granular and all processes will incorrectly be identified as being new.
```

Comment:
"Too coarse identifier" should be "Too coarse an identifier" ; "detect new process effectively" should be "detect new processes effectively."